tests/tests_sycl/2d_examples/test_2d_column_collapse_sycl/column_collapse_sycl.cpp

```diff
@@ -0,0 +1,266 @@
+
+    Relation<Inner<>> soil_block_inner(soil_block);
+    Relation<Contact<>> soil_block_contact(soil_block, {&wall_boundary});
+
+    UpdateRelation<MainExecutionPolicy, Inner<>, Contact<>> soil_block_update_complex_relation(soil_block_inner, soil_block_contact);
+    ParticleSortCK<MainExecutionPolicy, RadixSort> particle_sort(soil_block);
+    //----------------------------------------------------------------------
+    //	Define the main numerical methods used in the simulation.
+    //	Note that there may be data dependence on the constructors of these methods.
+    //----------------------------------------------------------------------
+    Gravity gravity(Vecd(0.0, -gravity_g));
+    StateDynamics<MainExecutionPolicy, GravityForceCK<Gravity>> constant_gravity(soil_block, gravity);
+    StateDynamics<execution::ParallelPolicy, NormalFromBodyShapeCK> wall_boundary_normal_direction(wall_boundary);
+    StateDynamics<MainExecutionPolicy, fluid_dynamics::AdvectionStepSetup> soil_advection_step_setup(soil_block);
+    StateDynamics<MainExecutionPolicy, fluid_dynamics::AdvectionStepClose> soil_advection_step_close(soil_block);
+
+
+    InteractionDynamicsCK<MainExecutionPolicy, continuum_dynamics::PlasticAcousticStep1stHalfWithWallRiemannCK>
+         soil_acoustic_step_1st_half(soil_block_inner, soil_block_contact);
+    InteractionDynamicsCK<MainExecutionPolicy, continuum_dynamics::PlasticAcousticStep2ndHalfWithWallRiemannCK>
+        soil_acoustic_step_2nd_half(soil_block_inner, soil_block_contact);
+    InteractionDynamicsCK<MainExecutionPolicy, fluid_dynamics::DensityRegularizationComplexFreeSurface>
+        soil_density_regularization(soil_block_inner, soil_block_contact);
+    InteractionDynamicsCK<MainExecutionPolicy, continuum_dynamics::StressDiffusionInnerCK> stress_diffusion(soil_block_inner);
+    ReduceDynamicsCK<MainExecutionPolicy, fluid_dynamics::AcousticTimeStepCK> soil_acoustic_time_step(soil_block,0.4);
+    //----------------------------------------------------------------------
+    //	Define the methods for I/O operations, observations
+    //	and regression tests of the simulation.
+    //----------------------------------------------------------------------
+    BodyStatesRecordingToVtp body_states_recording(sph_system);
+    body_states_recording.addToWrite<Vecd>(wall_boundary, "NormalDirection");
+    body_states_recording.addToWrite<Real>(soil_block, "Density");
+    StateDynamics<MainExecutionPolicy,continuum_dynamics::VerticalStressCK> vertical_stress(soil_block);
+    body_states_recording.addToWrite<Real>(soil_block, "VerticalStress");
+    StateDynamics<MainExecutionPolicy,continuum_dynamics::AccDeviatoricPlasticStrainCK> accumulated_deviatoric_plastic_strain(soil_block);
+    body_states_recording.addToWrite<Real>(soil_block, "AccDeviatoricPlasticStrain");
+    RestartIO restart_io(sph_system);
+    RegressionTestDynamicTimeWarping<ReducedQuantityRecording<TotalMechanicalEnergy>>
```

Comment:
you forget run this on device. Pleas refer
https://github.com/Xiangyu-Hu/SPHinXsys/blob/b56f01fb4392aa9a6ecdcc53ec925c7f08811d6a/tests/2d_examples/2d_examples_ck/test_2d_dambreak_ck/dambreak_ck.cpp#L132